WRO-630: Add support for SCSS/Sass (#103)
enactjs/dev-utils: CHANGELOG.md

# unreleased

> * Added `sass|scss` expansion at fileIdentPattern in `css-module-ident` module.

Reviewer: ```suggestion
* `css-module-ident`: Added `sass` and `scss` file extension to `fileIdentPattern`.
```
